### PR TITLE
ERR_002 | freestyleProject.cy.js | TC-001-1|Delete a project

### PR DESCRIPTION
**Pull request by katy1313**
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/86
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/94

### CODE REVIEW

File: cypress/e2e/deleteFreestyleProject.cy.js [hunk 1]
```diff
@@ -1,23 +1,19 @@
 /// <reference types="cypress"/>
 
-describe('US_001|Delete a project', ()=>{
-    it('TC-001-1|Create a project by clicking on "Create a job"', () => {
+describe('US_01.004 | FreestyleProject > Delete Project', ()=>{
+    it('TC_00.001.04 A|FreestyleProject > Delete Project from the dashboard', () => {
         cy.get('span').contains('Create a job').click();
         cy.wait(500);
```
Comment (Negrescu29) [inline cypress/e2e/deleteFreestyleProject.cy.js:6]: @katy1313 does this `cy.wait(500)`  aka 0.5 seconds have any sense here?

File: cypress/e2e/deleteFreestyleProject.cy.js [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
-    })
-
-    it.skip('TC-001-1|Delete a project', ()=>{
+        cy.get('a').contains("Dashboard").click();
         cy.get('span').contains('New Freestyle project').scrollIntoView();
         cy.get('span').contains('New Freestyle project').realHover();
         cy.get('button[data-href="http://localhost:8080/job/New%20Freestyle%20project/"]').click();
         cy.get('button[href="/job/New%20Freestyle%20project/doDelete"]').click();
         cy.get('button.jenkins-button.jenkins-button--primary ').click();
         cy.get('span').contains('New Freestyle project').should('not.exist');
```
Comment (Negrescu29) [inline cypress/e2e/deleteFreestyleProject.cy.js:17]: @katy1313 I run this in cypress and for the assertion you have on line 17, cypress says the `span` is undefined.

Screenshot 2024-11-18 at 23 41 42

If I understand the logic here you want to check if the item is not shown anymore in the list after you delete it? we had a similar case in our group - you're looking at a span which in this case is the list but after you delete the item there is no list shown, just the **welcome to Jenkins** screen 




Screenshot 2024-11-18 at 23 48 04

I believe you can check against the welcome screen that the list is not show that means the item was deleted

**Review by Negrescu29 — CHANGES_REQUESTED** [review]
left comments

**Review by Negrescu29 — CHANGES_REQUESTED** [review]
@katy1313 sorry just saw your spec file name, it should be the same name as the user story name. here is an example https://redroverschool.slack.com/archives/C049HHDG0KF/p1731979320828209?thread_ts=1731956893.790899&cid=C049HHDG0KF

**Review by Negrescu29 — APPROVED** [review]
👍🏼

**Review by Rom-T — APPROVED** [review]

**Pull request merged**